### ISSUE TITLE
Change div width: Uncaught TypeError: Cannot read property 'getSegments' of undefined resource

### ISSUE THREAD

**Issue by aerwin** [maqetta/maqetta#3515]
Using Chrome on Mac and local server running from Eclipse...
1. New HTML file
2. Refresh browser to get rid of any potential lingering effects of #3513
3. Drag and drop div from widget palette onto page
4. Go to props palette and try to change width to something like `34px`
5. Change fails and see following in console:

```
Uncaught TypeError: Cannot read property 'getSegments' of undefined resource.js:269
Resource.findResource resource.js:269
declare._canModifyRule Cascade.js:108
declare._addClasses Cascade.js:566
declare._buildCssRuleset Cascade.js:511
declare._updateCascadeList Cascade.js:590
declare._widgetSelectionChanged Cascade.js:1116
declare._updatePaletteValues SwitchingStyleView.js:319
(anonymous function) SwitchingStyleView.js:638
(anonymous function) lang.js:375
target.(anonymous function).dispatcher aspect.js:94
declare.onButtonClick StackController.js:257
(anonymous function) StackController.js:105
(anonymous function) lang.js:375
```

and then:

```
_onFieldChange. this._targetValueIndex is not a number Cascade.js:155
```

**Comment by JonFerraiolo** [comment]
Closing. I believe this was fixed as part of 8c69e58a35a18af1a6f908db10aaf6330d1da401

@aerwin - could you please verify that this is fixed?

**Comment by aerwin** [comment]
@JonFerraiolo Yes, appears to be fixed. Thanks!
